cmd/tempo-vulture/main.go

```diff
@@ -262,6 +263,7 @@ func pushMetrics(metrics traceMetrics) {
 	metricTracesErrors.WithLabelValues("incorrectresult").Add(float64(metrics.incorrectResult))
 	metricTracesErrors.WithLabelValues("missingspans").Add(float64(metrics.missingSpans))
 	metricTracesErrors.WithLabelValues("notfound_search").Add(float64(metrics.notFoundSearch))
+	metricTracesErrors.WithLabelValues("notfound_search").Add(float64(metrics.notFoundTraceQL))
```

Comment:
we should change the reason label from `notfound_search` to `notfound_traceql` or something